src/web/mjs/connectors/Tapas.mjs

```diff
@@ -47,24 +47,34 @@ export default class Tapas extends Connector {
 
     async _getChapters(manga) {
         let chapterList = [];
+
         let request = new Request(new URL(manga.id, this.url), this.requestOptions);
-        let data = await this.fetchDOM(request, 'div.paging a.paging__button--num');
-        let pageCount = data.length === 0 ? 1 : parseInt(data.pop().href.match(/pageNumber=(\d+)/)[1]);
-        for(let page = 1; page <= pageCount; page++) {
-            let chapters = await this._getChaptersFromPage(manga, page);
-            chapterList.push(...chapters);
+        let data = await this.fetchDOM(request, 'meta[name="twitter:app:url:iphone"]');
+        const series_id = data[0].content.match(/((?:\/series\/)\d+)/)[0];
```

Comment:
Regex may be simplified to `match(/\/series\/(\d+)/)[1]` without the non-capturing group